app.js

```diff
@@ -0,0 +1,104 @@
+var Issue = can.DefineMap.extend({
+  seal: false
+}, {
+  id: 'number',
+  title: 'string',
+  sort_position: 'number',
+  body: 'string'
+});
+
+Issue.List = can.DefineList.extend({
+  '*': Issue
```

Comment:
I got this from the chat guide JS Bin, so I’ll update it to have the new syntax. Should we show a deprecation warning for the old syntax?